provider/pkg/schema/gen.go

```diff
@@ -892,11 +917,30 @@ func addUntypedPropDocs(propertySpec *pschema.PropertySpec, cfTypeName string) {
 	}
 }
 
+// reportMissingDocs add missing docs to the report
+func (ctx *cfSchemaContext) reportMissingDocs(desc, propName string) {
+	if desc == "" {
+		val, ok := ctx.reports.MissingDocs[ctx.cfTypeName]
+		if ok {
+			if _, ok := val[propName]; !ok {
+				val[propName] = "1"
+			}
+		} else {
+			ctx.reports.MissingDocs[ctx.cfTypeName] = map[string]string{
+				propName: "1",
+			}
+		}
+	}
+}
+
 func (ctx *cfSchemaContext) propertySpec(propName, resourceTypeName string, spec *jsschema.Schema) (*pschema.PropertySpec, error) {
 	typeSpec, err := ctx.propertyTypeSpec(naming.LowerAcronyms(propName), spec)
 	if err != nil {
 		return nil, err
 	}
+
+	ctx.reportMissingDocs(spec.Description, propName)
```

Comment:
nit: It might be a bit surprising that it only reports the missing docs if description is empty.

But that's very nit and just my own preference so feel free to ignore

Reply:
I think that is the only place that docs are, unless I am missing something.